src/critics/path_align_critic.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright 2022 @artofnothingness Alexey Budyakov, Samsung Research
+#include "mppic/critics/path_align_critic.hpp"
+
+#include <xtensor/xfixed.hpp>
+#include <xtensor/xmath.hpp>
+
+namespace mppi::critics
+{
+
+void PathAlignCritic::initialize()
+{
+  auto getParam = parameters_handler_->getParamGetter(name_);
+  getParam(power_, "path_align_cost_power", 1);
+  getParam(weight_, "path_aling_cost_weight", 3.0);
+
+  getParam(path_point_step_, "path_point_step", 1);
+  getParam(trajectory_point_step_, "trajectory_point_step", 2);
```

Comment:
Still missing `trajectory_point_step` from readme

Reply:
oh missed that.. added